src/object/spawnpoint.hpp

```diff
@@ -25,6 +25,13 @@
 class ReaderMapping;
 class DrawingContext;
 
+namespace {
+ /**
+  * The spawnpoint that gets activated at the start of a game session)
+  */
+  const std::string DEFAULT_SPAWNPOINT_NAME = "main";
+}
+
```

Comment:
Wouldn't it be better to declare `DEFAULT_SPAWNPOINT_NAME` as `extern` in header file and define it in source file? Currently this will cause that identifier getting duplicated for each translation unit.